### PR TITLE
chore: bump fedimint-tonic-lnd version

### PR DESCRIPTION
**Pull request by okjodom**
- The new version of fedimint-tonic-lnd supports connecting to LND nodes using IP addresses

### CODE REVIEW

File: sim-lib/Cargo.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ serde = { version="1.0.183", features=["derive"] }
 serde_json = "1.0.104"
 bitcoin = { version = "0.30.1", features=["serde"] }
 lightning = { version = "0.0.116" }
-tonic_lnd = { package="fedimint-tonic-lnd", version="0.1.0", features=["lightningrpc", "routerrpc"]}
+tonic_lnd = { package="fedimint-tonic-lnd", version="0.1.1", features=["lightningrpc", "routerrpc"]}
 tonic = { version = "0.8", features = ["tls", "transport"] }
```
[the remark this answers is not in the document]
Comment (sr-gi) [reply sim-lib/Cargo.toml:20]: What's the motivation for `0.10` over any other and what are the advantages of going from `0.8` to `0.10`?

File: sim-lib/src/lnd.rs [hunk 2]
```diff
@@ -197,10 +197,10 @@ impl LightningNode for LndNode {
                 let payment = stream.map_err(|err| LightningError::TrackPaymentError(err.to_string()))?;
                 match payment {
                     Some(payment) => {
-                        let payment_status = PaymentStatus::from_i32(payment.status)
-                            .ok_or(LightningError::TrackPaymentError("Invalid payment status".to_string()))?;
-                        let failure_reason = PaymentFailureReason::from_i32(payment.failure_reason)
-                            .ok_or(LightningError::TrackPaymentError("Invalid failure reason".to_string()))?;
+                        let payment_status: PaymentStatus =payment.status.try_into()
```
Comment (sr-gi) [inline sim-lib/src/lnd.rs:200]: nit: there's a missing space there

File: sim-lib/Cargo.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (okjodom) [reply sim-lib/Cargo.toml:20]: I picked the latest stable versions of tonic when bumping `fedimint-tonic-lnd` in hopes we can stay there for a long while.
Think this was a lack of foresight to match the version of tonic `cln-grpc`, but we would not be able to sustain such a match of versions anyway.

[the remark this answers is not in the document]
Comment (sr-gi) [reply sim-lib/Cargo.toml:20]: Agreed, I don't think that's realistic

**Comment by okjodom** [comment]
update: using opportunity to apply a less strict semver in tonic_lnd dependency so we can just keep rolling on the patch version

**Comment by okjodom** [comment]
update: cargo update to fetch the latest version of `fedimint-tonic-lnd`, now [available in crates.io](https://crates.io/crates/fedimint-tonic-lnd/versions)

**Comment by okjodom** [comment]
update: avoided cargo update in favor of more atomic version bump :p

**Review by sr-gi — APPROVED** [review]
tACK [bcd3b2f](https://github.com/bitcoin-dev-project/sim-ln/pull/151/commits/bcd3b2ffa6c4da9f6946099345bd8869bfad1f8b)

**Review by carlaKC — APPROVED** [review]
tACK, only briefly reviewed the upstream change but it lgtm!

**Comment by carlaKC** [comment]
Would be nice to tack on a change to the readme that has IPs because it's still pretty odd to need `https://ip address` - either push with this PR or in a followup, I'm easy.

**Comment by okjodom** [comment]
> Would be nice to tack on a change to the readme that has IPs because it's still pretty odd to need `https://ip address` - either push with this PR or in a followup, I'm easy.

appended in readme with ip/domain addresses in examples. we always need https for these tls connections.

**Review by carlaKC — APPROVED** [review]
🚢  ittttt

**Pull request merged**